frontend/src/main/components/SectionsTableBase.js

```diff
@@ -37,19 +37,19 @@ export default function SectionsTableBase({ columns, data, testid = "testid"}) {
                     <td
                     {...cell.getCellProps()}
                     data-testid={`${testid}-cell-row-${cell.row.index}-col-${cell.column.id}`}
-                    // Stryker disable next-line ObjectLiteral
+                    // Stryker disable all
```

Comment:
Why are we disabling all here?

All of this looks very testable.

Stryker disable is only for cases of very complex logic (such as the interaction of the various caches inside of the "react-query" library, and local storage. (Even local storage and react-query we now have solutions for, so technically, there should be no *new* code that falls under stryker exceptions.).

Please remove this and let's see if we can deal with whatever Stryker is throwing at us.  This is just sweeping it under the rug.